1-3.md

```diff
@@ -21,27 +21,30 @@ navigation:
  
 Courses based on this unit:
 * Explore the rationale for organizations to address accessibility.
-* Present different testimonials from organizations that have addressed accessibility.
+* Present examples of organizations addressing accessibility.
```

Comment:
Not sure we need to focus on "testimonials" only. There can be good examples of organizations addressing accessibility without testimonials. Also, "have addressed" sounds past-tense while accessibility is an ongoing process.